.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 30 days with no activity.'
+          days-before-stale: 30
+          days-before-close: 5
+          # Never close a PR
+          days-before-pr-close: -1
+          # We should only close issues that we have explicitly answered/interacted with
+          any-of-labels: 'invalid,needs-more-info,answered'
```

Comment:
This means that we auto close issues labeled with: `invalid`, `needs-more-info`, or `answered`.

I think its better to mark the issues as needing stale checks, as opposed to doing it to all of them.